src/Components/test/E2ETest/ServerExecutionTests/ComponentHubReliabilityTest.cs

```diff
@@ -119,6 +120,7 @@ public async Task CannotInvokeJSInteropBeforeInitialization()
         }
 
         [Fact]
+        [Repeat(1000)]
```

Comment:
Am I holding this right?

Reply:
I think so. Is this just to figure out issues with the test or do we plan on checking in something like this?

Reply:
I think it works if you put it on the class too?

Reply:
The tests passed, so I took it out